src/Menu/MenuManager.php

```diff
@@ -103,10 +109,12 @@ protected function resolve(): void
                 data_get($this->resolved, $path) ?? [],
                 [
                     'label' => $label ?? Str::afterLast($path, '.'),
-                    'uri' => Str::of($resolvedRoute->uri())->start('/')->toString(),
+                    'uri' => filter_var($resolvedRoute->uri, FILTER_VALIDATE_URL)
```

Comment:
please do $resolvedRoute->uri() instead of $resolvedRoute->uri, to keep it consistent.

Reply:
only one of the two occurrences updated